### PR TITLE
Doc: Rephrase managed cluster requirement for clarification

### PR DESCRIPTION
**Pull request by yue9944882**
> Signed-off-by: yue9944882 <291271447@qq.com>

### CODE REVIEW

File: content/en/getting-started/quick-start/_index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ __Contents__
 ## Prerequisites
 
 - The hub cluster should be `v1.19+`.
-- At least one managed cluster greater than `v1.16+`.
+- The managed clusters should be `v1.16+`.
```
Comment (qiujian16) [inline content/en/getting-started/quick-start/_index.md:22]: I think we support cluster higher than v1.11 today. But it is recommended to use version higher then v1.16

Comment (yue9944882) [reply content/en/getting-started/quick-start/_index.md:22]: oh i didnt know about that before. will the registration controller automatically falling back to v1beta1 crd?

Comment (qiujian16) [reply content/en/getting-started/quick-start/_index.md:22]: hub still needs to be 1.19+, and we only require hub cluster to have csr v1 API. There are some workaround for lease API in registration, and support v1beta1 crds in registration-operator.

Comment (yue9944882) [reply content/en/getting-started/quick-start/_index.md:22]: > There are some workaround for lease API in registration, and support v1beta1 crds in registration-operator.

how you think about clarifying the workaround in the document?

Comment (qiujian16) [reply content/en/getting-started/quick-start/_index.md:22]: let's merge this one at first. I will add more here to indicate how to support earlier kube version

**Review by qiujian16 — COMMENTED** [review]
/lgtm

**Pull request merged**